model/src/main/proto/spine_examples/shareaware/watchlist/commands.proto

```diff
@@ -26,12 +26,12 @@
 
 syntax = "proto3";
 
-package spine_examples.shareaware.command;
+package spine_examples.shareaware.watchlist.command;
```

Comment:
Please update the Proto packages according to [this comment](https://github.com/spine-examples/ChatSPN/pull/3#discussion_r1129424448).